src/App.tsx

```diff
@@ -1,18 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
-
 export const App: React.FC = () => {
+  const [selectedPage, setSelectPage] = useState(1);
+  const [itemsOnPage, setItemsOnPage] = useState(5);
+
+  const [totalItems] = useState(42);
```

Comment:
Why do you need useState for this value? Move this constant outside the component at all